src/Sulu/Bundle/CustomUrlBundle/Resources/public/js/components/webspace/settings/custom-url/input/main.js

```diff
@@ -13,7 +13,7 @@ define(['underscore', 'jquery', 'services/husky/util'], function(_, $, util) {
 
     var defaults = {
             templates: {
-                input: '<input type="text" class="form-element<% if (!!prefix) { %> prefix" data-prefix="true"<% } else { %>" data-suffix="true"<% } %>/>'
+                input: '<div class="input-part"><input type="text" class="form-element<% if (!!prefix) { %> prefix" data-prefix="true"<% } else { %>" data-suffix="true"<% } %>/></div>'
```

Comment:
Is that `div`-wrapper required? Or can you set the CSS properties also to `input`?